Apps/APN-UIKit/APN UIKit/APN UIKit/View/DashboardViewController.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+class DashboardViewController: UIViewController {
+
+    static func newInstance() -> DashboardViewController {
+        UIStoryboard.getViewController(identifier: "DashboardViewController")
+    }
+    
+    var dashboardRouter: DashboardRouting?
+    
+    override func viewWillAppear(_ animated: Bool) {
+         super.viewWillAppear(animated)
+        self.navigationController?.isNavigationBarHidden = true
```

Comment:
What about moving this into an extension function of `UIViewController` to make it re-usable? 

Then, we can have something like this: 
```suggestion
        self.hideNavigationBar()
```